authentik/policies/geoip/countries.py

```diff
@@ -0,0 +1,253 @@
+"""List of all ISO-3166-1 alpha-2 country codes"""
```

Comment:
An alternative to this adding a dependency like [iso3166](https://github.com/deactivated/python-iso3166) or [django-countries](https://github.com/SmileyChris/django-countries).

Since ISO 3166 changes _infrequently_, I erred on the side of fewer dependencies.

Reply:
I don't mind adding a dependency for this, especially since we should add gettext calls to all these humanized names, and some dependency might have full translation for all the country names already

Reply:
Added `django-countries`. I'm unfamiliar with `drf-spectacular` and needed a bit of dark magic to make it work. If there's an easier way, let me know.